README.md

```diff
@@ -1,36 +1,77 @@
+[![Build Status](https://travis-ci.org/SOM-Timer/som_timer_fe.svg?branch=main)](https://travis-ci.org/SOM-Timer/som_timer_fe)
+[![Contributors][contributors-shield]][contributors-url]
+[![Issues][issues-shield]][issues-url]
+[![Issues][issues-closed-shield]][issues-closed-url]
+
 # Som Timer
```

Comment:
What if we put in 
```markdown
<div align="center"><img src="src/assets/content/meditationContentIcon.png" alt="woman sitting in meditation pose"/>
<img src="src/assets/content/somaticContentIcon.png" alt="human head with arrows connecting to body"/>
<img src="src/assets/content/yogaContentIcon.png" alt="woman in warrior yoga pose"/></div>

 # <div align="center">Som Timer</div>
```

where the Title is?

Reply:
I tried popping those in there & adding a height of 30px to each image. What do you think? 
<img width="674" alt="Screen Shot 2020-11-03 at 10 08 35 PM" src="https://user-images.githubusercontent.com/62770765/98071578-2f39c000-1e21-11eb-9900-bcd91a4c4bd2.png">

Reply:
I love this!!!! Adding the icons is SO the move!

Will the line colors turn dark on a light background? I think GitHub default is white!